internal/datastore/postgres/migrations/manager.go

```diff
@@ -1,15 +1,99 @@
 package migrations
 
 import (
+	"context"
+	"embed"
+	"io"
+	"path"
+	"strings"
+
 	"github.com/authzed/spicedb/pkg/migrate"
 
 	"github.com/jackc/pgx/v4"
 )
 
+const (
+	dir = "migrations"
+
+	createVersionTable = `CREATE TABLE alembic_version (
+    		version_num VARCHAR NOT NULL
+		);
+		INSERT INTO alembic_version (version_num) VALUES ('');`
+)
```

Comment:
why is this bit not handled also into its own SQL file?

Reply:
If you're running migrations with an external tool, that external tool almost certainly has its own tracking table (and we can't just put all `alembic_version` modifications in the migration files, since they will be DML and a goal is to have some DDL-only files).

I was thinking of `spicedb migrate` as "just another tool" that can consume the migration files to run them; the `alembic_version` table is what it uses for bookkeeping.

Reply:
resolving since this is not relevant to the current implementation